src/lib/did.ts

```diff
@@ -1,16 +1,75 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import {findVerificationMethod as bazaarFindVerificationMethod, CachedResolver} from '@digitalbazaar/did-io';
-import didSol from '@identity.com/did-io-driver-sol';
 import {DIDDocument} from "did-resolver";
 import {IDiDResolver} from "./resolver";
+import {DidSolIdentifier, DidSolService} from "@identity.com/sol-did-client";
 
-const resolver = new CachedResolver();
+const resolveSolDid = (did: string): Promise<DIDDocument> => {
+    return DidSolService.build(
+        DidSolIdentifier.parse(did),
+    ).resolve()
+}
+
+/**
+ * Borrowed from https://github.com/digitalbazaar/did-io/blob/main/lib/did-io.js
+ */
+const VERIFICATION_RELATIONSHIPS = new Set([
+    'assertionMethod',
+    'authentication',
+    'capabilityDelegation',
+    'capabilityInvocation',
+    'keyAgreement'
+]);
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+function bazaarFindVerificationMethod({doc, methodId, purpose} = {}) {
```

Comment:
Could we create an interface for `{doc, methodId, purpose}` ? And then mark what is required and optional there? I think we can then remove the typeErrors cause we will move the type checking to compile?